src/widgetastic_patternfly/__init__.py

```diff
@@ -393,6 +393,68 @@ def __repr__(self):
         return '{}({!r})'.format(type(self).__name__, self.locator)
 
 
+class BootstrapNav(Widget):
+    """Encapsulate a Bootstrap nav component"""
```

Comment:
Maybe a couple more words about it, also where locator should point exactly, maybe a link to the component itself ...